################################################################################
SHORT DESCRIPTION: 
################################################################################
Only set variables if they are not set.

################################################################################
LONG DESCRIPTION: 
################################################################################
This prevents the values needed in the OpenXT build environment from being
overriden at build time.

################################################################################
CHANGELOG 
################################################################################

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################
No.

################################################################################
INTERNAL DEPENDENCIES 
################################################################################

################################################################################
PATCHES 
################################################################################
Index: xen-4.6.5/config/StdGNU.mk
===================================================================
--- xen-4.6.5.orig/config/StdGNU.mk
+++ xen-4.6.5/config/StdGNU.mk
@@ -1,13 +1,13 @@
 AS         = $(CROSS_COMPILE)as
-LD         = $(CROSS_COMPILE)ld
+LD         ?= $(CROSS_COMPILE)ld
 ifeq ($(clang),y)
 CC         = $(CROSS_COMPILE)clang
 CXX        = $(CROSS_COMPILE)clang++
 LD_LTO     = $(CROSS_COMPILE)llvm-ld
 else
-CC         = $(CROSS_COMPILE)gcc
+CC         ?= $(CROSS_COMPILE)gcc
 CXX        = $(CROSS_COMPILE)g++
-LD_LTO     = $(CROSS_COMPILE)ld
+LD_LTO     ?= $(CROSS_COMPILE)ld
 endif
 CPP        = $(CC) -E
 AR         = $(CROSS_COMPILE)ar
